app/controllers/tag_controller.rb

```diff
@@ -55,6 +55,15 @@ def show
     @wikis = nodes if @node_type == 'wiki'
     @nodes = nodes if @node_type == 'maps'
     @title = params[:id]
+    notes = Node.where(status: 1, type: 'note')
+                 .includes(:revision, :tag)
+                 .where('term_data.name = ?', params[:id])
+    users = notes.collect(&:author).uniq
+    if !users.first.nil?
```

Comment:
Hmm, i wonder if we could do this more compactly like:

 `@length = user.length || 0` ?

does that syntax work in Ruby?

Thanks for your contribution!!